### DISCUSSION TITLE
Portal / Webinterface

### DISCUSSION THREAD

**Discussion by geraldFS** [philibertc/micronova_controller#21] [Ideas]
Is it possible to use a web interface in which you can change some settings (similar to Tasmota)

The following settings would be very helpful:
- DHCP / IP address
- host name
- MQTT (server / port / user / password / topic) / sync interval
- Oven adjustments (Watertemp = var / flue gas temperature = 0x5A etc.)
- serial monitor?

I know a big board but totally awesome if it worked ...

And do not forget:
The program is awesome, it's wonderful!
Thanks for that!

**Comment by philibertc** [comment]
Hello, it is indeed in my improvement projects. Currently the web page is accessible only after a reset, which can be annoying.
The following parameters are already implemented:
- MQTT (server, port, user, pass, base topic)

Glad you like the project!
